server/src/__tests__/routes/twiml.test.ts

```diff
@@ -1,9 +1,9 @@
 import { createTwimlRoute } from '../../routes/twiml';
-import { twiml, validateRequest } from 'twilio';
+import { twiml, validateExpressRequest } from 'twilio';
 
 const mockedVoiceResponse = jest.mocked(twiml.VoiceResponse);
 
-jest.mocked(validateRequest);
+jest.mocked(validateExpressRequest);
```

Comment:
`jest.mocked` is a no-op utility function that just lets you get a `jest.Mock` out of a package, so the use case for this here is something like this

```
const mockedValidateExpressRequest = jest.mocked(validateExpressRequest);
```

and TS will give you things like

```
mockedValidateExpressRequest.mock.calls;
```

without complaints. Since we're not using the return value here, this `jest.mocked` call isn't really doing all that much.